doc/Docker_deployment.md

```diff
@@ -24,6 +24,17 @@ Note: this has only been tested in OS X.
 9. docker pull kbase/deplbase
 
 ## Deploying a module
-
-1. docker build -t mymodulename .
-2. docker run --name mymodulename -d mymodulename
+1. generate cluster.ini and ssl
+```shell
+     git clone https://github.com/kbaseincubator/deploy_dev
+     cd deploy_dev/
+     cp site.cfg.example site.cfg
+     vi site.cfg
```

Comment:
The documentation is using "kbase/deplbase" instead of "kbase/depl"
and the user may have additional change on their site configuration.

Thanks,
-SJ

On Wed, Nov 4, 2015 at 12:40 PM, Nomi Harris <notifications@github.com>
wrote:

> In doc/Docker_deployment.md
> <https://github.com/kbase/kb_sdk/pull/33#discussion_r43912544>:
>
> > @@ -24,6 +24,17 @@ Note: this has only been tested in OS X.
> >  9. docker pull kbase/deplbase
> >
> >  ## Deploying a module
> > -
> > -1. docker build -t mymodulename .
> > -2. docker run --name mymodulename -d mymodulename
> > +1. generate cluster.ini and ssl
> > +```shell
> > +     git clone https://github.com/kbaseincubator/deploy_dev
> > +     cd deploy_dev/
> > +     cp site.cfg.example site.cfg
> > +     vi site.cfg
>
> What's this vi step about? What are they changing in site.cfg?
> I think this section needs more explanation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kbase/kb_sdk/pull/33/files#r43912544>.
>